server/routes/health_routes.py

```
@@ -120,82 +117,14 @@ async def send_chatbot_test_message():
-def check_chatbot_websocket():
-    """Check chatbot WebSocket service by sending a test message."""
     try:
-        return asyncio.run(send_chatbot_test_message())
-    except Exception as e:
-        logger.warning(f"Chatbot health check failed: {e}")
-        return {"status": "unhealthy", "error": "Chatbot WebSocket health check failed"}
+        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        sock.settimeout(3)
+        sock.connect((host, port))
+        sock.close()
+        return {"status": "healthy", "message": f"Chatbot accepting connections on {host}:{port}"}
```

Ruff (0.15.13)

[warning] Consider moving this statement to an `else` block

(TRY300)